### DISCUSSION TITLE
copilot enabled and online but no completions

### DISCUSSION THREAD

**Discussion by flyjgh** [community/community#15864] [Copilot Conversations] [answered]
hi, I use the nvim copilot extension since a few months, but the last few days, the competions doesn't happen anymore even though Copilot's status shows "enabled and online".
I tried `:Copilot restart` but I still have no completions.
i'm not sure where the problem is coming from.
Any help would be greatly appreciated

**Comment by davecheney** [comment, marked as answer]
@flyjgh please try updating

**Reply by flyjgh** [reply]
Thanks, I already have the latest version

**Reply by davecheney** [reply]
What I can see is you are using version 1.7.9, 1.7.29 is the latest neovim release.

**Reply by flyjgh** [reply]
thanks for your time but I don't get what you're reffering to, unless I'm mistaking, the latest version of copilot in nvim is 1.2.2, and the latest nvim version is 0.7.0

**Reply by davecheney** [reply]
Yes. Sorry it’s confusing, neovim copilot 1.2.2 contains version 1.7.29 of the copilot agent code. When I looked at your telemetry it appeared to be coming from a codebase that was using version 1.7.9 of the copilot agent codebase.

**Reply by flyjgh** [reply]
thanks a lot for your help, everything is upgraded, but I'm also using copilot on vscode so maybe that's what you are reffering to? Is there any nvim plugins known to interfer with copilot, or init.vim parameters?

**Comment by dan-cooke** [comment]
Having the same issue on 

```
copilot.vim 1.9.1
Neovim 0.9.1
dist/agent.js 1.94.0
Node.js 18.15.0
```

When i run `:Copilot panel` it says 

`Not authentcated : Not signed in`

When I run `:Copilot auth` it says i am successfully authenticated
